src/neural/writer.h

```diff
@@ -35,8 +35,9 @@ namespace lczero {
 
 #pragma pack(push, 1)
 
-struct V4TrainingData {
+struct V5TrainingData {
   uint32_t version;
+  uint32_t input_format;
```

Comment:
Inserting this at the beginning is annoying for me in rescorer and training scripts, but it puts the float probabilities at an 8 byte offset that could potentially make for faster memcopies, and it kind of naturally belongs with the version field. I think I can survive the downsides :P